app/javascript/components/bootcamp/JikiscriptExercisePage/hooks/useConstructRunCode/useConstructRunCode.ts

```diff
@@ -122,33 +125,54 @@ export function useConstructRunCode({
           }
         }
       )
-      try {
-        testResults = generateAndRunTestSuite({
+      // try {
+      testResults = await generateAndRunTestSuite(
+        {
           studentCode,
           tasks,
           config,
           customFunctions: customFns,
-        })
-      } catch (error) {
-        console.log(error)
-        const compError = error as CompilationError
-        if (
-          compError.hasOwnProperty('type') &&
-          compError.type == 'CompilationError'
-        ) {
-          handleCompilationError(compError.error, editorView)
-          return
-        }
-        console.log(compError)
-      }
+        },
+        {
+          setUnderlineRange,
+          setHighlightedLine,
+          setHighlightedLineColor,
+          setShouldShowInformationWidget,
+          setInformationWidgetData,
+        },
+        editorView
+      )
+      // console.log("Thinks I've run", testResults.tests.length)
+      // } catch (error) {
+      //   console.log(error)
+      //   const compError = error as CompilationError
+      //   if (
+      //     compError.hasOwnProperty('type') &&
+      //     compError.type == 'CompilationError'
+      //   ) {
+      //     handleCompilationError(compError.error, editorView)
+      //     return
+      //   }
+      //   console.log(compError)
+      // }
```

Comment:
@iHiD can we delete this?

Reply:
No, we need it. You can delete the console logs, but the rest is important for the Jikiscript version.